src/librustc/ty/mod.rs

```diff
@@ -1818,6 +1847,14 @@ impl<'tcx> TyS<'tcx> {
             }
         }
     }
+
+
+    pub fn get_repr(&self) -> Option<&ReprOptions> {
```

Comment:
Yeah, it could. But my thought is that things like future lints are going to want it as well.